topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorTest.java

```diff
@@ -995,6 +1008,90 @@ public void testReconcileAllTopics_listMapsFails(VertxTestContext context) {
         }));
     }
 
+    @Test
+    public void testFailedReconcileMetrics(VertxTestContext context) throws InterruptedException {
```

Comment:
Rather than adding specific tests for the metrics, did you consider adding assertions on the metrics to the existing tests in TOT? That would provide better coverage.